Homework1/BWT/BWT.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace BWT;
+
+using System; 
+using System.Text;
+
+// Class that contains a BWT transform implementation.
+internal static class BWT
+{
+    static string Rotate(String oldString)
```

Comment:
```suggestion
    static string Rotate(string oldString)
```
Ну либо везде использовать имена классов, а не ключевые слова, их обозначающие. По мне так лучше ключевые слова.